cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/appusageevents/ApplicationUsageEventEntity.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2013-2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.cloudfoundry.client.v2.appusageevents;
+
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+
+/**
+ * The entity response payload for the Application Usage Event resource
+ */
+@Data
+public final class ApplicationUsageEventEntity {
+
+    /**
+     * The application id
+     *
+     * @param applicationId the application id
+     * @return the application id
+     */
+    private final String applicationId;
+
+    /**
+     * The application name
+     *
+     * @param applicationName the application name
+     * @return the application name
+     */
+    private final String applicationName;
+
+    /**
+     * The buildpack id
+     *
+     * @param buildpackId the buildpack id
+     * @return the buildpack id
+     */
+    private final String buildpackId;
+
+    /**
+     * The buildpack name
+     *
+     * @param buildpackName the build pack name
+     * @return the buildpack name
+     */
+    private final String buildpackName;
+
```

Comment:
The links in the issues don't help (they were made before we had the `latest-release` URI alias, but there are a couple of fields missing because you were looking at `214` instead of `232` the latest.  I'll add them in and just try to keep in mind that you want to look at the "latest" version of the documentation.